webpack.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
         include: path.join(__dirname, 'src', 'assets'),
         loader: 'url-loader',
         options: {
-          limit: 8192, // in bytes
+          limit: 8192,
```

Comment:
@NkFab Why did you remove that comment? I think that it is useful because only having that limit does not tell you if it is in Mb's, Kb's and so on.

Reply:
That makes sense.